Arguments shadow builtin names

After we have updated `flake8-builtins` we got new several new errors.

```
=================================== FAILURES ===================================
_________________________________ FLAKE8-check _________________________________
/home/travis/build/lk-geimfari/mimesis/mimesis/providers/business.py:47:5: A002 "copyright" is used as an argument and thus shadows a python builtin, consider renaming the argument
_________________________________ FLAKE8-check _________________________________
/home/travis/build/lk-geimfari/mimesis/mimesis/providers/cryptographic.py:37:5: A002 "hash" is used as an argument and thus shadows a python builtin, consider renaming the argument
/home/travis/build/lk-geimfari/mimesis/mimesis/providers/cryptographic.py:50:5: A002 "bytes" is used as an argument and thus shadows a python builtin, consider renaming the argument
```

--- a/mimesis/providers/business.py
+++ b/mimesis/providers/business.py
@@ -27,8 +27,7 @@ class Business(BaseDataProvider):
         :Example:
             Gamma Systems.
         """
-        return self.random.choice(
-            self._data['company']['name'])
+        return self.random.choice(self._data['company']['name'])
 
     def company_type(self, abbr: bool = False) -> str:
         """Get a random type of business entity.
@@ -40,11 +39,10 @@ class Business(BaseDataProvider):
             Incorporated.
         """
         return self.random.choice(
-            self._data['company']['type'].get(
-                'abbr' if abbr else 'title'),
+            self._data['company']['type'].get('abbr' if abbr else 'title'),
         )
 
-    def copyright(self) -> str:
+    def copyright(self) -> str:  # noqa: A002
         """Generate a random copyright.
 
         :return: Copyright of company.

--- a/mimesis/providers/cryptographic.py
+++ b/mimesis/providers/cryptographic.py
@@ -34,7 +34,7 @@ class Cryptographic(BaseDataProvider):
         bits = self.random.getrandbits(128)
         return str(uuid.UUID(int=bits, version=version))
 
-    def hash(self, algorithm: Optional[Algorithm] = None) -> str:
+    def hash(self, algorithm: Optional[Algorithm] = None) -> str:  # noqa: A002
         """Generate random hash.
 
         :param algorithm: Enum object ``Algorithm``.
@@ -47,7 +47,7 @@ class Cryptographic(BaseDataProvider):
             fn = getattr(hashlib, key)
             return fn(self.uuid().encode()).hexdigest()
 
-    def bytes(self, entropy: int = 32) -> Bytes:
+    def bytes(self, entropy: int = 32) -> Bytes:  # noqa: A002
         """Generate byte string containing *entropy* bytes.
 
         The string has *entropy* random bytes, each byte

